omniNotes/src/main/java/it/feio/android/omninotes/utils/date/ReminderPickers.java

```diff
@@ -62,11 +66,20 @@ public void onDateTimeRecurrenceSet(SelectedDate selectedDate, int hourOfDay, in
         Calendar reminder = selectedDate.getFirstDate();
         reminder.set(Calendar.HOUR_OF_DAY, hourOfDay);
         reminder.set(Calendar.MINUTE, minute);
-
-        mOnReminderPickedListener.onReminderPicked(reminder.getTimeInMillis());
-        mOnReminderPickedListener.onRecurrenceReminderPicked(
-            RecurrenceHelper
-                .buildRecurrenceRuleByRecurrenceOptionAndRule(recurrenceOption, recurrenceRule));
+        long reminderTimeInMillis = reminder.getTimeInMillis();
```

Comment:
No need to have these two variables in Java heap if they're used just here. Why not to change the whole block to something like:
```java
...
  if (reminder.getTimeInMillis() < System.currentTimeMillis()) {
    Snackbar.make(mActivity.findViewById(android.R.id.content), "Cannot set alarm in the past", Snackbar.LENGTH_LONG).show();
    return;
  }

  mOnReminderPickedListener.onReminderPicked(reminder.getTimeInMillis());
  mOnReminderPickedListener.onRecurrenceReminderPicked(
          RecurrenceHelper.buildRecurrenceRuleByRecurrenceOptionAndRule(recurrenceOption, recurrenceRule));
}
```
Less code, easier to read.